src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    @Override
+    public List<Excavator> get() {
+        List<Excavator> excavators = new ArrayList<>();
+        for (int i = 0; i < AMOUNT_OF_MODELS; i++) {
```

Comment:
Ensure that the `AMOUNT_OF_MODELS` constant is defined and has access modifier. If it's not defined, it will cause a compilation error. If it's public or protected, consider making it private if it's only used within this class.